src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -0,0 +1,146 @@
+		self.sess.run(tf.compat.v1.global_variables_initializer())
+
+		# Assign the initialised weights from MXNet to tensorflow 
+		layers = ['drawing_conv0_weight', 'drawing_conv0_bias', 'drawing_conv1_weight', 'drawing_conv1_bias', 'drawing_conv2_weight', 'drawing_conv2_bias', 'drawing_dense0_weight', 'drawing_dense0_bias', 'drawing_dense1_weight', 'drawing_dense1_bias']
+		for key in layers:
+			if 'bias' in key:
+			    self.sess.run(_tf.compat.v1.assign(_tf.compat.v1.get_default_graph().get_tensor_by_name(key+":0"), net_params[key].data().asnumpy()))
+			else:
+			    if 'dense' in key:
+			        self.sess.run(_tf.compat.v1.assign(_tf.compat.v1.get_default_graph().get_tensor_by_name(key+":0"), net_params[key].data().asnumpy().transpose(1,0)))
+			    else:
+			        self.sess.run(_tf.compat.v1.assign(_tf.compat.v1.get_default_graph().get_tensor_by_name(key+":0"), net_params[key].data().asnumpy().transpose(2, 3, 1, 0)))
+
+
+	def tf_train_model(self, train_loader, validation_loader, validation_set, verbose):
+		num_iter = 0
+		if verbose:
+			column_names = ['iteration', 'train_loss', 'train_accuracy', 'time']
+			column_titles = ['Iteration', 'Training Loss', 'Training Accuracy', 'Elapsed Time (seconds)']
+			if validation_set is not None:
+			    column_names.insert(3, 'validation_accuracy')
+			    column_titles.insert(3, 'Validation Accuracy')
+			table_printer = _ProgressTablePrinter(column_names, column_titles)
+
+		def process_data(batch_data):
+			if batch_data.pad is not None:
+				batch_x = batch_data.data[0].asnumpy().transpose(0, 2, 3, 1)[0:self.batch_size-batch_data.pad]
+				batch_y = tf.keras.utils.to_categorical(batch_data.label[0].asnumpy()[0:self.batch_size-batch_data.pad], self.num_classes)
+			else:
+				batch_x = batch_data.data[0].asnumpy().transpose(0, 2, 3, 1)
+				batch_y = tf.keras.utils.to_categorical(batch_data.label[0].asnumpy(), self.num_classes)
+			return batch_x, batch_y
+
+		start_time = _time.time()
+		for train_batch in train_loader:
+			batch_x, batch_y = process_data(train_batch)
+			_, c, a = self.sess.run([self.optimizer, self.cost, self.accuracy], 
+			                feed_dict={
+			                    self.x: batch_x, 
+			                    self.y: batch_y
+			                })
+			for val_batch in validation_loader:
+				val_x, val_y = process_data(val_batch)
+				val_acc = self.sess.run(self.accuracy, 
+			                feed_dict={
+			                    self.x: val_x, 
+			                    self.y: val_y
+			                })
+			validation_loader.reset()
+			if verbose:
+				kwargs = {  "iteration": num_iter + 1,
+				            "train_loss": "{:.3f}".format(c),
+				            "train_accuracy": "{:.3f}".format(a),
+				            "time": _time.time() - start_time}
+				if validation_set is not None:
+				    kwargs["validation_accuracy"] = "{:.3f}".format(val_acc)
+				table_printer.print_row(**kwargs)
+			num_iter+=1
+
+		final_train_accuracy = a
```

Comment:
`final_train_accuracy` and `final_train_loss` are way better names than `a` and `c`. Why not just always use the better name (i.e. remove these two lines and change line 116)?